### ISSUE TITLE
dh_make in stubbs:archive can be invoked non-interactively

### ISSUE THREAD

**Issue by eedwardsav** [rerun/rerun#245]
I'm debugging a build issue and for whatever reason in the environment I'm building, this line creates a problem:

```
    DEBFULLNAME="rerun-discuss@googlegroups.com"; \
    yes | dh_make --indep --copyright apache --email rerun-discuss@googlegroups.com
```

dh_make should support a `-y` option to invoke it non-interactively

so this issue is to track the possibility of changing to that invocation style

**Comment by stagrlee** [comment]
Silly request, would you mind changing -y to --yes?

**Comment by eedwardsav** [comment]
>Silly request, would you mind changing -y to --yes?

Done :)

**Comment by eedwardsav** [comment]
Closed by #246
